docusaurus.config.js

```diff
@@ -68,6 +68,14 @@ const config = {
             to: 'https://docs.bpftrace.org/latest',
             from: '/docs',
           },
+          {
+            to: 'https://docs.bpftrace.org/next',
+            from: '/docs/master',
```

Comment:
should be `/docs/master.html`.

Reply:
Damn, that path works on my local server but not in prod for some reason. It also doesn't like 'docs/master.html' as the 'from'. I'll see if I can find another solution.

Reply:
Ok, should work now. I made some static pages that redirect.

Reply:
Sweet, thanks!